src/deadline/maya_adaptor/MayaClient/render_handlers/vray_handler.py

```diff
@@ -63,6 +63,29 @@ def start_render(self, data: dict) -> None:
             raise RuntimeError("MayaClient: start_render called without a valid camera.")
         self.render_kwargs["camera"] = self.camera_name
 
+        # In order of preference, use the task's output_file_prefix, the step's output_file_prefix, or the scene file setting.
+        output_file_prefix = data.get("output_file_prefix", self.output_file_prefix)
+        if output_file_prefix:
+            maya.cmds.setAttr(
+                "defaultRenderGlobals.imageFilePrefix", output_file_prefix, type="string"
+            )
+
+        if self.image_width is not None:
+            maya.cmds.setAttr("vraySettings.width", self.image_width)
+            print(f"Set image width to {self.image_width}", flush=True)
+        if self.image_height is not None:
+            maya.cmds.setAttr("vraySettings.height", self.image_height)
+            print(f"Set image height to {self.image_height}", flush=True)
+
+        region = [
+            data.get(field)
+            for field in ("region_min_x", "region_max_x", "region_min_y", "region_max_y")
+        ]
+        if any(v is not None for v in region):
+            raise RuntimeError(
+                "MayaClient: A region render was specified, but region rendering support is not implemented for the selected renderer."
+            )
```

Comment:
Would it make sense to put this in a common function since it's copied a few times?

Reply:
Yeah, there's more that could be done to clean up how the implementation is split between the base class and each renderer-specific subclass. I think that will best be done as a separate PR.